invenio_app_ils/ui/src/invenio_app_ils/common/api/urls.js

```diff
@@ -1,5 +1,6 @@
 export const ApiURLS = {
   loans: {
+    email: '/circulation/loans/:loanPid/email-overdue',
```

Comment:
small suggestion, even here `email` -> `emailOverdue`

Reply:
`.../:loanPid/emailOverdue` like this?

Reply:
oh the other way, sure, `emailOverdue: '...'`